### DISCUSSION TITLE
How can I get a price from y coordinate, where the crosshair points?

### DISCUSSION THREAD

**Discussion by newtilus** [tradingview/lightweight-charts#1102] [Q&A] [answered]
Hi there,

I struggle with one thing - how can I get a price from a specific **y** coordinate, not related to my series, from outside of the range? I would like to click on the chart and save the price selected by the user, which is different then the price on the chart.
I'm trying to use `subscribeCrosshairMove` and `subscribeClick` like below, but it will only give me the price range of the actual candle in the x position. Not the price from the pricescale of the **y** position. So in the example below I need to get **0,13** price, right where the crosshair is.
<img width="782" alt="Zrzut ekranu 2022-06-30 o 23 51 09" src="https://user-images.githubusercontent.com/684260/176785443-ca48b39a-0aa1-4816-ac42-d0674efd1aa3.png">

`chart.subscribeClick((param) => {
var price = param.seriesPrices.get(candleSeries);
var coordinatePrice = candleSeries.coordinateToPrice(2);
currentDataDiv.innerHTML = JSON.stringify(param)+" "+JSON.stringify(price);
});
`

**Comment by edew** [comment, marked as answer]
@newtilus 

You can use https://tradingview.github.io/lightweight-charts/docs/api/interfaces/ISeriesApi#coordinatetoprice

For example
```
const series = chart.addLineSeries();
series.setData([ /* some data */ ]);

const myClickHandler = (param) => {
  if (!param.point) {
    return;
  }

  console.log(`The price is ${series.coordinateToPrice(param.point.y)}`);
}

chart.subscribeClick(myClickHandler);
```

**Reply by newtilus** [reply]
Thanks for the answer, it works! I'm extremely surprise, as I thought that as it addresses the series, it will take the price from the series as well, not from the crosshair position! Cheers!

**Reply by newtilus** [reply]
For future reference, maybe somebody will find it usefull, here's the updated code:

 ```
   chart.subscribeClick((param) => {
        var price = param.seriesPrices.get(candleSeries);
        var coordinatePrice = candleSeries.coordinateToPrice(param.point.y);
        log("The price is: "+coordinatePrice);
    });
```

**Comment by GiacomoR90** [comment]
ok with this function do I receive the price data in the coordinates?  What if I need the time?
